samples/ironside_se/secondary_boot_gen_uicr/secondary/prj.conf

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# NB: Must match secondary_partition in DT
+CONFIG_FLASH_LOAD_OFFSET=0x1b0000
```

Comment:
Could we have a static assert ensuring this? Not critical as this is just a sample, but I am curious as to how an incorrect configuration would be reported.

Reply:
fixed (moved to DT)